### PR TITLE
Fix menu styles

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-09-28 at 15 57 58

### CODE REVIEW

File: src/components/seeds/database/index.tsx [hunk 1]
```diff
@@ -478,29 +481,16 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
-              sx={{ ':hover': { background: 'none' } }}
-              id='edit-columns'
-            >
-              <Icon name='iconColumns' />
-
-              <Typography sx={{ color: '#136BD3', paddingLeft: 1 }}>{strings.CUSTOMIZE_TABLE_COLUMNS}</Typography>
-            </IconButton>
-          </Box>
-        </Box>
+        <MenuList sx={{ padding: theme.spacing(2, 0) }}>
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:484]: remove spacing here?

File: src/components/seeds/database/index.tsx [hunk 2]
```diff
@@ -478,29 +481,16 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
+          <MenuItem onClick={() => onDownloadReport()} id='download-report' sx={{ padding: theme.spacing(1, 2) }}>
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:485]: add more vertical spacing here and below?

File: src/components/seeds/database/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/seeds/database/index.tsx:484]: Removing parent padding and adding to children looks like this:

Screen Shot 2022-09-28 at 16 17 25

Is that correct?

Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:484]: hmm..my mistake. looks like your original version is consistent with our menus

Screen Shot 2022-09-28 at 12 22 53 PM

**Review by karthikbtf — APPROVED** [review]
original fix looks good 🎉

**Pull request merged**